### ISSUE TITLE
Textures

### ISSUE THREAD

**Issue by Hungry-123** [Sushi8756/Cookie-Clicker-Source-Code#1]
There are quite a few missing textures in the website, and the Krumblor Cookie Dragon "Dragon Harvest","Dragon Flight", and the option for the "Supreme Intellect" abilities like the dragon abilities for the mini-games are not appearing. If you find what's wrong and fix it that would be great.

**Comment by Sushi8756** [comment]
So it is just the textures for the buttons to select the cookie dragon ability? Or is it the ability itself?

**Comment by Hungry-123** [comment]
The textures for the idleverses, and Cortex Bakers are not there for when muted, or when in the store. Also, the dragon ability for the mini-games affects are not there, and little dragon icon is not there. And the dragon harvest and Dragon Flight cookie spawns do not seem to appear as soon as it should when slotted in the dragon ability.

**Comment by Sushi8756** [comment]
OK, I will try to fix the textures, but I can't do anything about the cookie spawns, sorry about that.

**Comment by Hungry-123** [comment]
Thanks, but the dragon abilites also don't show up in the building minigames. I'll upload a picture as an example. 
[https://imgur.com/a/4orfhsT](https://imgur.com/a/4orfhsT)

**Comment by Hungry-123** [comment]
That was an example of how it's supposed to look like, but it normally doesn't show up.

**Comment by DiscretionIsAdvised** [comment]
For me the big cookie is gone, as well as the textures for the idleverses, cortex bakers, a lot of the milk flavors, and the santa test tube.

**Comment by Sushi8756** [comment]
> For me the big cookie is gone, as well as the textures for the idleverses, cortex bakers, a lot of the milk flavors, and the santa test tube.

OK is that on a local download? If it is then try re-downloading it. If it is online then just try again, maybe clear the cache. Those textures are there for me.

**Comment by Sushi8756** [comment]
Alright I'm done procrastinating, can you give me a save file that has the dragon building minigames and dragon flight cookie so I can get the texture files?

**Comment by Hungry-123** [comment]
Yeah, i'll be honest, i stopped playing cookie clicker a while ago, but i'll put a save file that doesn't show the textures on your website.

**Comment by Hungry-123** [comment]
[The69ersBakery(1).txt](https://github.com/Sushi8756/Cookie-Clicker-2.048/files/10254439/The69ersBakery.1.txt)

**Comment by Sushi8756** [comment]
Thanks I will get on that when I get home.

**Comment by Sushi8756** [comment]
> Thanks I will get on that when I get home.

Lmao still haven't done it 💀
